src/python/WMCore/ReqMgr/CherryPyThreads/StepChainParentageFixTask.py

```diff
@@ -65,8 +67,9 @@ def fixStepChainParentage(self, config):
         totalChildDS = len(childDatasets)
         fixCount = 0
         for childDS in childDatasets:
+            self.logger.info("Resolving parentage for dataset: %s", childDS)
             start = int(time.time())
-            failedBlocks = self.dbsSvc.fixMissingParentageDatasets(childDS, insertFlag=True)
+            failedBlocks = self.dbsSvc.fixMissingParentageDatasets(childDS, insertFlag=False)  # FIXME: set to True
```

Comment:
set to True and remove the comment please ;)

Reply:
Done :)